teams/wg-rustc-reading-club.toml

```diff
@@ -0,0 +1,16 @@
+name = "wg-rustc-reading-club"
+subteam-of = "compiler"
+kind = "working-group"
+
+[people]
+leads = ["nikomatsakis", "doc-jones"]
+members = ["doc-jones", "nikomatsakis"]
+
+[website]
+name = "Rust Code Reading Club working group"
+description = "Helping new and experienced contributors learn more about rustc."
+repo = "https://rust-lang.github.io/rustc-reading-club/"
+zulip-stream = "rustc-reading-club"
+
+[[github]]
+orgs = ["rust-lang"]
```

Comment:
nit: I think you removed one too many blank lines; usually it's good to leave a blank line at the end of files. That's why GitHub put a red mark here.

Reply:
Yeah, I actually didn't intend to remove both.  I don't it's critical enough to prevent a successful review, though.